docs/hub/models-gated.md

```diff
@@ -1,94 +1,149 @@
+<div class="flex justify-center">
+    <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-enabled.png"/>
+    <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-enabled-dark.png"/>
+</div>
 
-```
----
-extra_gated_prompt: "You agree to not use the model to conduct experiments that cause harm to human subjects."
-extra_gated_fields:
- Company: text
- Country: text
- I agree to use this model for non-commercial use ONLY: checkbox
----
-```
+If you want to manually approve which users can access your model, you must set it to **manual approval**. When this is the case, you will notice more options:
+- **Add access** allows you to search for a user and grant them access even if they did not requested it.
+- **Notification frequency** let you configure when to get notified if new users request access. It can be set to once a day or real-time. By default, an email is sent to your primary email address. You can set a different email address in the **Notifications email** field. For models hosted under an organization, emails are sent to the first 5 admins of the organization.
 
 <div class="flex justify-center">
-<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated.png"/>
-<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-dark.png"/>
+    <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-manual-approval.png"/>
+    <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-manual-approval-dark.png"/>
 </div>
 
-The `README.md` file for a model is called a [model Card](./model-cards). Visit the documentation to learn more about how to use it and to see the properties that you can configure.
+### Review access requests
 
-## Manual approval
+Once access requests are enabled, you have full control of who can have access to your model or not, no matter if the approval mode is set to manual or automatic. You can review and manage requests either from the UI or via the API.
 
-By default, requests to access the model are automatically accepted.
-Model authors can set the approval mode to "Manual reviews" from the model's **Settings** tab.
-Doing so enforces that each access request will be manually reviewed and approved by the model authors.
-Only users whose access requests have been approved will be able to access the model's content.
+### From the UI
+
+You can review who have access to your gated model from its setting page by clicking on the **Review access requests** button. This will open a modal with 3 lists of users:
```

Comment:
```suggestion
You can review who has access to your gated model from its settings page by clicking on the **Review access requests** button. This will open a modal with 3 lists of users:
```